client/src/components/Main/NoteArea.tsx

```diff
@@ -15,13 +21,14 @@ const NoteArea = ({ activeNote, updateNote }: NoteAreaProps) => {
     );
   }
 
-  if (!activeNote?.title) activeNote.title = "";
-  if (!activeNote?.body) activeNote.body = "";
+  if (!activeNote.title) activeNote.title = "";
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/zerexei/zoop-note/pullRequest?prid=7539925)
```suggestion
  if (!activeNote.title) {activeNote.title = "";}
```